inc/Action/Diff.php

```diff
@@ -17,7 +17,7 @@ class Diff extends AbstractAction
     /** @inheritdoc */
```

Comment:
https://github.com/dokuwiki/dokuwiki/blob/df992f97cb02d677b035f6035cb1515b2ae3a3d1/inc/Action/AbstractAction.php#L42

Needs to be updated as well.

Reply:
updated.